### PR TITLE
New Nukie Gun, The Zorg ZF-1 Pod Weapon System

### PR DESCRIPTION
**Pull request by MajManatee**
# Document the changes in your pull request

Gives the nukies a new weapon to play with when killing everyone, the Zorg ZF-1 Pod Weapon System from the movie The Fifth Element

Features include:
3 Round burst, 300-round capacity.
Rocket Launcher
Bolt Launcher
Net Thrower
Flamethrower
and (if I can figure it out)
Ice Cube Freezing system (Cold thrower)
Maybe add the self-destruct like a mini bomb?

Cost is TBD, and stats are still up in the air and able to be changed

- [ ] make it not fire bullets when firing another weapon
- [ ] icecube system
- [ ] self destruct
- [ ] sprites
- [x] actually giving it to the nukies
- [x] Net gun in other mode maybe?
- [ ] the flamethrower has to function
- [x] Make the magazine not removable

# Spriting
No solid sprites are done yet, but this is a general idea. Mako is working on them
image

# Wiki Documentation

New nukie gun, and related information

# Changelog

:cl:  MajManatee, @AlvCyktor
wip: This is work in progress  
rscadd: Badass new gun for the nukies, the Zorg ZF-1 Pod Weapon System
imageadd: Sprites for the gun
/:cl:

### CODE REVIEW

**Comment by KillerOrcaCora** [comment]
The weed you are smoking is literally just shavings of highly radioactive uranium...keep it up

**Comment by MusselMain** [comment]
I love a movie reference, but guddam.

**Comment by MusselMain** [comment]
Also had to mention that maybe it should self destruct if anyone aside from nuke ops or brainwashed tries to use it.

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 1]
```diff
@@ -212,6 +212,74 @@
 	burst_size = 3
 	fire_delay = 1
 
+/obj/item/gun/ballistic/automatic/zf1
+	name = "Zorg ZF-1 Pod Weapon System"
+	desc = "The ZF-1 is an ambidextrous multifunctional weapon incapable of being detected by X-Rays featuring a titanium recharger, 300 round clip with a 3 round burst. It also features a variety of other fire modes as well, including a self-destruct button."
```
Comment (Anvilman6) [inline code/modules/projectiles/guns/ballistic/automatic.dm:217]: Massively bloated wordage here - be concise

File: code/modules/uplink/uplink_items.dm [hunk 2]
```diff
@@ -574,6 +574,14 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 25
 	include_modes = list(/datum/game_mode/nuclear, /datum/game_mode/infiltration)
 
+/datum/uplink_item/dangerous/sniper
+	name = "Zorg ZF-1 Pod Weapon System"
+	desc = "The ZF-1 is an ambidextrous multifunctional weapon incapable of being detected by X-Rays featuring a titanium recharger, 300 round clip with a 3 round burst. It also features a variety of other fire modes as well, including a self-destruct button."
```
Comment (Anvilman6) [inline code/modules/uplink/uplink_items.dm:579]: Same as above

**Review by Anvilman6 — CHANGES_REQUESTED** [review]
Draft review

File: code/modules/uplink/uplink_items.dm (hunk 2 above)
Comment (MajManatee) [reply code/modules/uplink/uplink_items.dm:579]: The quote is pulled directly from the movie when describing the weapon, hence its original inclusion

Comment (Anvilman6) [reply code/modules/uplink/uplink_items.dm:579]: Perhaps try to make it not a 1:1 rip

Comment (MajManatee) [reply code/modules/uplink/uplink_items.dm:579]: to be fair, it is already shorter, but I get it.

**Comment by maxion12345** [comment]
Icecube could just shoot a glueon grenade

**Comment by Bibby0110** [comment]
Stale, looks cool

**Pull request closed without merging**